starters/spring-web-starters/README.md

```diff
@@ -0,0 +1,19 @@
+# Spring Web Examples
+
+The started Spring Boot Web Application  will be available at `localhost:5050`.
+
```

Comment:
🔨 Could you give a more elaborate description what is happening in the spring-web-starter? Also in regards to CloudEvents and the Gateway.

Reply:
This was actually moved as is from gateway-c7 but i can add some documentation here

Reply:
That would be super helpful. Ideally someone who opens up the readme knows right aways what's going on in here. :)